patch/kconfig-inclusions

```diff
@@ -126,6 +126,62 @@ CONFIG_SENSORS_EMC2305=m
 [marvell-arm64]
 CONFIG_MARVELL_AC5_WATCHDOG=m
 
+[pensando]
+CONFIG_ARCH_PENSANDO=y
+CONFIG_ARCH_PENSANDO_ELBA_SOC=y
+CONFIG_PENSANDO_SOC_CAPMEM_HUGEPAGE=y
+CONFIG_PENSANDO_SOC_PENFW=y
+CONFIG_EDAC_ELBA=y
+CONFIG_IRQ_PENSANDO=y
+CONFIG_MFD_PENSANDO_ELBASR=y
+CONFIG_RESET_ELBASR=y
+CONFIG_UIO_PCIEMAC=m
+CONFIG_UIO_PENMSI1=m
+CONFIG_UIO_PENGIC=m
+CONFIG_UIO_PENMSI=m
+CONFIG_ARM64_VA_BITS_39=y
+CONFIG_COMMON_CLK_CS2000_CP=y
+CONFIG_CONTEXT_TRACKING=y
+CONFIG_CRYPTO_MANAGER_DISABLE_TESTS=y
+CONFIG_DEFAULT_SECURITY_DAC=y
+CONFIG_DEVTMPFS_MOUNT=y
+CONFIG_EMBEDDED=y
+CONFIG_EXT3_FS=y
```

Comment:
Why does the ext3 filesystem code need to be enabled here? ext4 can handle ext2 and ext3.

Reply:
This can be removed.

Reply:
New commit added for this.